README.md

```diff
@@ -5,28 +5,26 @@ Welcome to the SQL Server Language Extensions project!
 
 Starting in SQL Server 2019, we have added support for language extensibility, which means that you can now execute code in various languages like R, Python and Java from SQL Server.
 
+To download the released prebuilt versions of these extensions, go to the [Releases section of this repository](https://github.com/microsoft/sql-server-language-extensions/releases). Download the latest version of whichever extension corresponds to your machine configuration (Windows or Linux). There are separate releases for each language extension.
+
 We hope you enjoy using Microsoft SQL Server and the language extension feature.
 
 -SQL Server Team
 
-## Announcements
-
-* In SQL Server 2019, we are open sourcing the SDK for our Java language extension: [**Microsoft Extensibility SDK for Java for Microsoft SQL Server​**](https://github.com/microsoft/sql-server-language-extensions/tree/master/language-extensions/java/sdk)
+## Documentation
 
-## Things to Know
+You can read more about how to use language extensions and how to use Java in SQL Server 2019 in the Microsoft [documentation](https://docs.microsoft.com/en-us/sql/language-extensions/language-extensions-overview?view=sqlallproducts-allversions).
 
-* The Java components in this project are built on Java 8
-* The SDK is also installed as part of SQL Server 2019 on both Windows as Linux
-  * Default installation path on Windows: [instance installation home directory]\MSSQL\Binn\mssql-java-lang-extension.jar
-  * Default installation path on Linux: /opt/mssql/lib/mssql-java-lang-extension.jar 
+Tutorials and documentation for the R and Python extensions can be found here:
 
-## Documentation
+[Python Documentation](https://docs.microsoft.com/en-us/sql/machine-learning/install/custom-runtime-py?view=sql-server-ver15)

```

Comment:
looks like this link has been modified to:
custom-runtime-py is renamed to custom-runtime-python
  
https://docs.microsoft.com/en-us/sql/machine-learning/install/custom-runtime-python?view=sql-server-ver15

we might need to fix it in Python readme too.